tests/inputFields.spec.ts

```diff
@@ -5,91 +5,84 @@ test.beforeEach( async({page}) => {
   //1. Select the PET TYPES menu item in the navigation bar
   await page.getByText('Pet Types').click()
   //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
-  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+  expect(await page.locator('h2').getByText('Pet Types').textContent()).toEqual('Pet Types')
 })
 
 test('Update pet type', async ({page}) => {
-  test.slow()
   //3. Click on "Edit" button for the "cat" pet type
   //assumes cat is always first
   await page.getByText('Edit').first().click()
 
   //4. Add assertion of the "Edit Pet Type" text displayed
-  expect(await page.getByText('Edit Pet Type').isVisible()).toBeTruthy()
+  expect(await page.getByText('Edit Pet Type').textContent()).toEqual('Edit Pet Type')
 
   //5. Change the pet type name from "cat" to "rabbit" and click "Update" button
-  const textField = page.locator('#name')
-  await textField.click()
-  await textField.clear()
-  await textField.pressSequentially('rabbit')
+  const editPetTypeField = page.locator('#name')
+  await editPetTypeField.click()
+  await editPetTypeField.clear()
+  await editPetTypeField.fill('rabbit')
   await page.locator('button').getByText('Update').click()
   
   //6. Add the assertion that the first pet type in the list of types has a value "rabbit" 
-  test.slow()
-  const rabbit = page.locator('input[name=pettype_name]').first().inputValue()
-  expect(await rabbit).toBe('rabbit')
-  
+  expect(await page.locator('input[name=pettype_name]').first().inputValue()).toEqual('rabbit')
   
   //7. Click on "Edit" button for the same "rabbit" pet type
   await page.getByText('Edit').first().click()
   
   //8. Change the pet type name back from "rabbit" to "cat" and click "Update" button
-  await textField.click()
-  await textField.clear()
-  await textField.pressSequentially('cat')
+  await editPetTypeField.click()
+  await editPetTypeField.clear()
+  await editPetTypeField.fill('cat')
   await page.getByText('Update').click()
   
   //9. Add the assertion that the first pet type in the list of names has a value "cat" 
-  const catEntry = page.locator('input[name=pettype_name]').first().inputValue()
-  expect(await catEntry).toBe('cat')
+  const updatedPetType = page.locator('input[name=pettype_name]').first().inputValue()
+  expect(await updatedPetType).toBe('cat')
```

Comment:
Updating of the name is better, but you still have duplications with the code above. And, you still using two lines of code instead of single line expression with Locator Assertion